### PR TITLE
Dependency updates week of 2023 04 24

### PR DESCRIPTION
**Pull request by ujahio**
### This round of updates includes:
- Publishing a new ECR image 2.26.0 which will require publishing this on the court side 🏀
- Major update to `stylelint-config-standard-scss` and `patch-package`.
- Minor updates
- [Deploy with migration](https://app.circleci.com/pipelines/github/flexion/ef-cms/43613/workflows/1f1ffbe3-5d49-405c-a5ae-df693c78fde3)

### CODE REVIEW

**Review by mmarcotte — APPROVED** [review]

File: scripts/verify-terraform-version.sh [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 tf_version=$(terraform --version)
 
-if [[ ${tf_version} != *"1.4.5"* ]]; then
+if [[ ${tf_version} != *"1.4.6"* ]]; then
   echo "Please set your terraform version to 1.4.5 before deploying."
```
Comment (rachelschneiderman) [inline scripts/verify-terraform-version.sh]: Nit: I think this should say 1.4.6 now 😄

Comment (ujahio) [reply scripts/verify-terraform-version.sh]: Hey @rachelschneiderman, I changed/updated the tf version to `1.4.6`. It does reflect that. I may be missing something..

Comment (rachelschneiderman) [reply scripts/verify-terraform-version.sh]: Apologies, that wasn't a super clear comment. I meant the echo statement on line 6 says to set your version to `1.4.5`, I think it should say `1.4.6` now!

Comment (ujahio) [reply scripts/verify-terraform-version.sh]: I understand now. Making the change.

**Review by codyseibert — APPROVED** [review]

**Review by rachelschneiderman — APPROVED** [review]

**Review by shumway-tm — APPROVED** [review]

**Review by rgoshen — APPROVED** [review]

**Review by TomElliottFlexion — APPROVED** [review]

**Review by jimlerza — APPROVED** [review]

**Pull request merged**